src/types/rule.ts

```diff
@@ -26,6 +26,10 @@ export interface IRule<T = string, O = any> {
   // Filter the target down to a subset[] with a JSON path
   given: string;
 
+  // If false, rule will operate on original (unresolved) data
```

Comment:
It seems like this will update our docs a bit? Could you add mention of it to the custom rules section?

Reply:
@philsturgeon added a short note, lmk what you think.